server/handlers.go

```diff
@@ -96,6 +97,7 @@ func (s *Server) discoveryHandler() (http.HandlerFunc, error) {
 		Keys:              s.absURL("/keys"),
 		UserInfo:          s.absURL("/userinfo"),
 		DeviceEndpoint:    s.absURL("/device/code"),
+		Introspect:        s.absURL("/introspect"),
```

Comment:
Let's make it `/token/introspect`, as in keycloak? I think introspect is more generic word.

Reply:
It requires changing `handleWithCORS("/introspect", s.handleIntrospect)` to `handleWithCORS("/token/introspect", s.handleIntrospect)` as well

Reply:
Done 💚